config/jobs/image-pushing/k8s-staging-e2e-test-images.sh

```diff
@@ -56,6 +56,11 @@ EOF
 for image in "${IMAGES[@]}"; do
     cat >>"${OUTPUT}" <<EOF
     - name: post-kubernetes-push-e2e-${image//\//-}-test-images
+      rerun_auth_configs:
+        github_users:
+          - aojea
+          - chewong
+          - claudiubelu
```

Comment:
Please also give rerun access to @kubernetes/test-infra-admins and @kubernetes/release-managers
```suggestion
      rerun_auth_configs:
        github_team_slugs:
          - org: kubernetes
            slug: release-managers
          - org: kubernetes
            slug: test-infra-admins            
        github_users:
          - aojea
          - chewong
          - claudiubelu
```